core/criteria/src/main/java/org/opennms/core/criteria/CriteriaBuilder.java

```diff
@@ -282,6 +283,24 @@ public CriteriaBuilder sql(final Object sql) {
         return this;
     }
 
+    public CriteriaBuilder sql(final Object sql, Object parameter, Type type) {
```

Comment:
This was a cut-and-paste from the original method... and I'm not sure why it was ever an ```Object```. I'll switch it to ```String```.